compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -884,45 +872,18 @@ class Namer { typer: Typer =>
      *  to pick up the context at the point where the completer was created.
```

Comment:
Could you please update this comment to say

Use the given `ctx` to complete this Completer.
Normally, this is just the creation context, as passed by `complete`.
However, if '-Yexplicit-nulls' is enabled, we pass in the current context so that flow typing works with blocks.